packages/api/api-generator/src/commands/generate/generateCrud/__tests__/generate-crud-position.spec.ts

```diff
@@ -1,13 +1,13 @@
+import { CrudGenerator } from "@comet/cms-api";
```

Comment:
I'm not convinced that it's a good idea to separate runtime and build-time code into two packages, but if @nsams is fine with it, it's okay for me.

Reply:
As we talked yesterday, we will keep this for now as it is, but we will try later to move all api-generator related stuff (Decorator)  into the @comet/api-generator package as well.